test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -246,25 +246,30 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 	It("should increase cluster size if pending pods are small and there is another node pool that is not autoscaled [Feature:ClusterSizeAutoscalingScaleUp]", func() {
 		framework.SkipUnlessProviderIs("gke")
 
-		By("Creating new node-pool with one n1-standard-4 machine")
+		By("Creating new node-pool with n1-standard-4 machines")
```

Comment:
For consistency, can we either edit it in all affected tests, or in none?

Reply:
Done.